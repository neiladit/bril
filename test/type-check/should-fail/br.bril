main{

    v1: int = const 2;
    v2: int = const 3;
    v3: int = add v1 v2;
    v4: int = const 6;
    v5: bool = eq v3 v4;
    br v4 here there;
    here:
    jmp there;
    there:
    print v4;
}
