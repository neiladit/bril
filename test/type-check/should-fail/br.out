	branch cond undefined properly
line 7: branch args undefined
br error!
